modules/apps/frontend-js/frontend-js-dependencies-web/package.json

```diff
@@ -35,6 +35,7 @@
 		"text-mask-core": "5.1.2",
 		"uuid": "9.0.0"
 	},
+	"main": "js/index.js",
```

Comment:
Is this really necessary? If yes, what's it for? AMD maybe? :thinking:

Reply:
It is necessary. I forget why exactly though...